v3_async_wip/v3_async_wip/transport_helper.py

```diff
@@ -0,0 +1,101 @@
+# -------------------------------------------------------------------------
```

Comment:
Just a helper for manual testing until E2E testing can be implemented. You can ignore this.